js/src/export_output.ts

```diff
@@ -13,4 +14,20 @@ AJAX.registerOnload('export_output.js', function () {
         e.preventDefault();
         $('#export_refresh_form').trigger('submit');
     });
+
+    $('.export_copy_to_clipboard_btn').on('click', function (e) {
+        e.preventDefault();
+        var textAreaText = $('textarea#textSQLDUMP').val();
+        var $temp = $('<textarea>');
+        $('body').append($temp);
+        $temp.val(textAreaText).trigger('select');
+        try {
+            var res = document.execCommand('copy');
+            $temp.remove();
+            ajaxShowMessage(window.Messages.strCopyQueryButtonSuccess, false, 'success');
+        } catch (e) {
+            $temp.remove();
+            ajaxShowMessage(window.Messages.strCopyQueryButtonFailure, false, 'error');
+        }
```

Comment:
```suggestion
            var res = Functions.copyToClipboard($('textarea#textSQLDUMP').val());
            if (res) {
                ajaxShowMessage(window.Messages.strCopyQueryButtonSuccess, false, 'success');
            } else {
                ajaxShowMessage(window.Messages.strCopyQueryButtonFailure, false, 'error');
            }
```

As I said on the issue, we already have code to copy to clipboard.
I made a suggestion, it might be not perfect but could you try it ?

It's inspired from:

https://github.com/phpmyadmin/phpmyadmin/blob/8355c0ce47d7ffbe0593f4019a33089b0e741895/js/src/modules/functions.ts#L1517